cmd/fleet/serve.go

```diff
@@ -548,6 +549,23 @@ the way that the Fleet server works.
 				appCfg.MDM.EnabledAndConfigured = true
 			}
 
+			// register the Microsoft MDM services
+			var (
+				wstepDepot *microsoft_mdm.WSTEPDepot
+			)
+			// TODO: check if MicrosoftMDM is enabled?
```

Comment:
This still needs to be harmonized with the feature flag approach

Reply:
you can use [IsMDMFeatureFlagEnabled](https://github.com/fleetdm/fleet/blob/e04e349bafac58c588b203255e6d939fe2e93adf/server/config/config.go#L1668) to check this. Usage example is [here](https://github.com/fleetdm/fleet/blob/e04e349bafac58c588b203255e6d939fe2e93adf/server/service/orbit.go#L225)

Reply:
@gillespi314 can you add this small change? I can add it later if needed, that's ok to me